url.js

```diff
@@ -0,0 +1,30 @@
+const { looksLikeLegacySSB, convertLegacySSB, parseUri } = require('ssb-fetch')
```

Comment:
Could these also be merged into `ssb.js`?

Reply:
Yes. Done. Although they ought to be in a package really; the omnibox in the ui code pulls this in too. I should make a separate package for this that ssb-fetch, the ui, and this extension can use. It's fine in here now to get this merged :)